packages/dashboard/src/pages/Wallets/components/FungibleTokenTable/index.tsx

```diff
@@ -93,6 +94,7 @@ export const FungibleTokenTable = memo<TokenTableProps>(({ selectedChainId }) =>
                         id: token.id,
                         name: token.name || '',
                         symbol: token.symbol || '',
+                        type: TrendingCoinType.Fungible,
```

Comment:
Introduce `TrendingCoinType` to distinguish between `Token` and `NFT`

Reply:
Can we use `TokenType`?